modules/LemDCBM400600/main/powermeterImpl.cpp

```diff
@@ -35,15 +40,26 @@ void powermeterImpl::ready() {
     // Start the live_measure_publisher thread, which periodically publishes the live measurements of the device
     this->live_measure_publisher_thread = std::thread([this] {
         this->publish_public_key_ocmf(this->controller->get_public_key_ocmf());
+        const Condition condition = Condition("powermeter/CommunicationFault", "Communication timed out", true);
```

Comment:
This looks a bit over complicated, maybe all you need is error_state_monitor->is_error_active() below?

Reply:
You're right, is_condition_satisfied calls is_error_active when used with one condition.
Fixed!